test/core/compiler.jl

```diff
@@ -77,12 +76,6 @@ priors = 0 # See "new grammar" test.
         chain = sample(fggibbstest(xs), gibbs, 2);
     end
     @testset "model macro" begin
-        model_info = Dict(:main_body_names => Dict(:vi => :vi, :sampler => :sampler))
-        # unit test model macro
-        expr = Turing.generate_observe(:x, :y, model_info)
-        @test expr.head == :block
-        @test :(vi.logp += Turing.observe(sampler, y, x, vi)) in expr.args
-
         @model testmodel_comp(x, y) = begin
```

Comment:
Can we add a test that only passing one variable throws errors here.

Reply:
Sure!